helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/AssignmentMetadataStore.java

```diff
@@ -50,33 +52,43 @@
   private Map<String, ResourceAssignment> _globalBaseline;
   private Map<String, ResourceAssignment> _bestPossibleAssignment;
 
+  AssignmentMetadataStore(String zkServers, String clusterName) {
+    this(new ZkBucketDataAccessor(zkServers), clusterName);
+  }
+
+  @VisibleForTesting
```

Comment:
This is actually a legitimate usage. As you could imagine, very soon we may encounter performance (latency) related problems, and the direction is to use a different metadata store instead of ZK.

That means we will have a different BucketDataAccessor implementation. Perhaps we should rename it to BlobDataAccessor or LargeDataAccessor.

So we should remove VisibleForTesting tag.